cluster/local/integration_tests_compute.sh

```diff
@@ -22,7 +22,7 @@ spec:
   forProvider:
     name: exampleIpBlock
     size: 2
-    location: es/vit
+    location: de/fra
```

Comment:
should we switch all e2e tests back to es/vit?

Reply:
at least the ones that rely on cpu family yes